wetmap/src/components/newModals/setting/index.tsx

```diff
@@ -0,0 +1,79 @@
+
+import SettingsView from './view';
+
+import { SessionContext } from '../../contexts/sessionContext';
+import { useContext, useState, useEffect } from 'react';
```

Comment:
useState, useEffect are both unused., please remove them

Reply:
I'm still seeing these being imported, why did you mark this as resolved with out making changes or at least documenting your reasoning in here as a response?